docs/js/main.js

```diff
@@ -13,6 +14,19 @@ import Root from './Root';
 import '../css/demo.scss';
 import '../img/favicon.ico';
 
+(() => injectGlobal`
+  ${styledNormalize}
```

Comment:
Just to make sure, this CSS applies only to the docs pages and won't be carried across to the users of these repo like `service-ui`, right?

Reply:
yes, it only applies to the docs page, however we will be using the workaround in `service-ui` as well